Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    metasploit-framework (6.0.44)
+    metasploit-framework (6.0.52)
```

Comment:
Looks like this isn't quite right still, it'd be good to the PR against master, and drop these changes :+1: